src/main.js

```diff
@@ -1,86 +1,64 @@
 import Container from "react-bootstrap/Container";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Card from "react-bootstrap/Card";
+import Button from "react-bootstrap/Button";
+
+class Main extends React.Component {
+
+
+  // updateSelected = (item) => this.props.updateSelected(item)
 
-class Main extends Component {
-  
   render() {
     return (
-      <Container>
-        <h1>Main is maining</h1>
-        <Row>
-          <Col>
-        <HornedBeasts
-          imgSrc={beastData[0].image_url}
-          alt={beastData[0].keyword}
-          title={beastData[0].title}
-          description={beastData[0].description}
-        />
-          </Col>
-          <Col>
-        <HornedBeasts
-          imgSrc={beastData[1].image_url}
-          alt={beastData[1].keyword}
-          title={beastData[1].title}
-          description={beastData[1].description}
-        />
-          </Col>
-          <Col>
-        <HornedBeasts
-          imgSrc={beastData[2].image_url}
-          alt={beastData[2].keyword}
-          title={beastData[2].title}
-          description={beastData[2].description}
-        />
-          </Col>
-        </Row>
-        <Row>
-          <Col>
-        <HornedBeasts
-          imgSrc={beastData[3].image_url}
-          alt={beastData[3].keyword}
-          title={beastData[3].title}
-          description={beastData[3].description}
-        />
-          </Col>
-          <Col>
-          </Col>
-          <Col>
-          </Col>
+      <Container fluid>
+        <Row md={3} lg={4} xl={5}>
+          {this.props.beastData.map((horn, idx) => (
+            <HornedBeasts
+              data={horn}
+              key={idx}
+              updateSelected={this.updateSelected}
+            />
+          ))}
         </Row>
       </Container>
     );
   }
 }
 
-class HornedBeasts extends Component {
-
+class HornedBeasts extends React.Component {
   constructor(props) {
-    super(props)
+    super(props);
 
     this.state = {
-      favorites: 0
-    }
+      favorites: 0,
+    };
   }
 
+  // updateSelected = (item) => this.props.updateSelected(item);
+
   render() {
     return (
-      <>
-        <h2>{this.props.title}</h2>
-        <Image
-          src={this.props.imgSrc}
-          alt={this.props.alt}
-          title={this.props.title}
-          onClick={() => this.setState({ favorites: this.state.favorites + 1 })}
+      <Card style={{ width: "15rem" }}>
+        <Card.Img variant="top" src={this.props.data.image_url} 
+        // onClick={this.updateSelected(this.props.image_url)}
         />
-        <p>&hearts; {this.state.favorites}</p>
-        <p>{this.props.description}</p>
-      </>
+        <Card.Body>
+          <Card.Title>{this.props.data.title}</Card.Title>
+          <Card.Text>{this.props.data.description}</Card.Text>
+          <Button
+            onClick={() =>
+              this.setState({ favorites: this.state.favorites + 1 })
+            }
+            variant="primary"
+          >
```

Comment:
You are on your way here, nice work so far!

I would remove this setState() call in the render as this can cause big issues. I might have commented on this before, but just calling attention to it again 👍